server/test/api/invoiceReceipt.js

```diff
@@ -13,12 +13,14 @@ describe('/reports/invoices Receipts Interface', function () {
   const validInvoice        = '957e4e79-a6bb-4b4d-a8f7-c42152b2c2f6';
   const invoiceItemLength   = 1;
   const invalidInvoice      = 'unknown';
-  const validRenderTarget   = '?render=json';
+  const jsonRenderTarget   = '?render=json';
```

Comment:
Just FYI, `superagent` (what `chai-http` is based on) accepts query parameters as an object, similar to angular.  I think it may be possible to do it with `chai-http`, if you think it looks nicer.  [Here is the relevant documentation](https://visionmedia.github.io/superagent/#query-strings).